src/components/sky-header/sky-header.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'sky-header',
+  templateUrl: 'sky-header.html',
+})
+export class SkyHeaderComponent implements OnInit {
+  @Input() title: string;
+  @Input() hours: number;
+  @Input() money: number;
+  @Input() secMoney: number;
```

Comment:
This naming does not really make sense, let's refer to the price per SKY as "price", and show the total value by simply multiplying price with amount in the template itself